io/CMakeLists.txt

```diff
@@ -5,13 +5,21 @@ else()
   add_definitions(-DNO_INLINE_POST)
 endif()
 
+if(PARALLEL_NETCDF)
+  set(NETCDF_PARALLEL_SRC model_write_netcdf_parallel.F90)
```

Comment:
Instead of maintaining separate stub file and maintaining changes in CMakeLists.txt and makefile wouldn't be easier to change module_write_netcdf_parallel.F90 to this structure (or something similar):
```
module module_write_netcdf_parallel

 ....

  private                                                                       
    public write_netcdf_parallel                                                  
                                                                                  
    contains                                                                      
                                                                                  
#ifndef NO_PARALLEL_NETCDF 

  <current content of module_write_netcdf_parallel.F90 subroutines>

#else

    subroutine write_netcdf_parallel(fieldbundle, wrtfb, filename, mpi_comm, mype, im, jm, rc)

     type(ESMF_FieldBundle), intent(in) :: fieldbundle
     type(ESMF_FieldBundle), intent(in) :: wrtfb
     character(*), intent(in)           :: filename
     integer, intent(in)                :: mpi_comm
     integer, intent(in)                :: mype
     integer, intent(in)                :: im, jm
     integer, optional,intent(out)      :: rc

     print *,'in write_netcdf_parallel - model not built with parallel netcdf support, return'

   end subroutine write_netcdf_parallel

#endif

end module module_write_netcdf_parallel
```

Reply:
Good idea - done.